mussel

```diff
@@ -727,6 +746,9 @@ printf "Optional pkg-config Support:    $PKG_CONFIG_SUPPORT\n"
 printf "Optional Quadmath Support:      $QUADMATH_SUPPORT\n"
 printf "Optional custom vendor:         '$CUSTOM_VENDOR'\n\n"
 
+# fail early if rsync is needed but not found
```

Comment:
As @firasuke said, if I'm not erred, rsync is not a dependency of kbuild anymore.